openapi/paths/objects@{object_id}.yaml

```diff
@@ -23,4 +23,47 @@ get:
       $ref: '../components/responses/500InternalServerError.yaml'
   tags:
     - Objects
-  x-swagger-router-controller: ga4gh.drs.server
\ No newline at end of file
+  x-swagger-router-controller: ga4gh.drs.server
+
+post:
+  summary: Get info about a DrsObject through POST'ing a Passport.
+  description: >-
+    Returns object metadata, and a list of access methods that can be used to fetch object bytes.
+
+    Method is a POST to accomodate a JWT GA4GH Passport sent in the formData in order to authorize access.
+  operationId: PostObject
+  security:
+    - PassportAuth: []
+  responses:
+    200:
+      $ref: '../components/responses/200OkAccess.yaml'
+    202:
+      $ref: '../components/responses/202Accepted.yaml'
+    400:
+      $ref: '../components/responses/400BadRequest.yaml'
+    401:
+      $ref: '../components/responses/401Unauthorized.yaml'
+    403:
+      $ref: '../components/responses/403Forbidden.yaml'
+    404:
+      $ref: '../components/responses/404NotFoundAccess.yaml'
+    500:
+      $ref: '../components/responses/500InternalServerError.yaml'
+  tags:
+    - Objects
+  x-swagger-router-controller: ga4gh.drs.server
+  parameters:
+    - $ref: '../components/parameters/ObjectId.yaml'
+    - $ref: '../components/parameters/Expand.yaml'
```

Comment:
I think having query string parameters in a POST request might violate some web conventions and/or standards. RESTful design generally favours using query string for GET requests, and request body (without any query string parameters) for POST requests (e.g. HTML forms with an action of `POST` take all parameters and submits them in the request body).

Perhaps we can have `expand` as a valid request body param rather than query string param, e.g.
```
{
  "expand": true,
  "tokens": [
    "...",
    "..."
  ]
}
```

Reply:
Want to talk about this on today's call?

Reply:
@briandoconnor forgot about this one at yesterday's call, only remembered now unfortunately. Is there any way we can solicit feedback on this item?